AccountService.js

```diff
@@ -3,9 +3,11 @@
  */
 'use strict';
 
-import axios from 'axios';
+import {httpClient} from '@digitalbazaar/http-client';
+import https from 'https';
```

Comment:
@JSAssassin this is browser code and you're pulling in nodejs stuffs.  An https agent is not required in browser code because it relies on the browser to provide all the workings around around self-signed certificates etc.  This is what is happening when we tell our browser to "accept the risk and continue" when it sees our development self-signed certificates.

Reply:
@mattcollier got it! I'll remove the agent then. Thanks